plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoClientConfig.java

```diff
@@ -59,6 +59,17 @@
     private String requiredReplicaSetName;
     private String implicitRowFieldPrefix = "_pos";
 
+    @AssertTrue(message = "Exactly one of these 'mongodb.seed' or 'mongodb.connection-url' must be specified")
+    public boolean isConnectionPropertyValid()
+    {
+        if (seeds.isEmpty() && (!connectionUrl.isPresent() || connectionUrl.get().isEmpty())) {
```

Comment:
`seeds.isEmpty() && connectionUrl.isEmpty()`

Reply:
This won't work. If the value is empty it will return false. It needs to be connectoinUrl.get().isEmpty(). 
isEmpty checks if the optional is null. It can't be null so it would have to be empty